.github/workflows/check-plugin-docs.js

```diff
@@ -0,0 +1,63 @@
+const fs = require("fs");
+const path = require("path");
+const fetchPluginDocs = require("../../website/components/remote-plugin-docs/utils/fetch-plugin-docs");
+
+const COLOR_RESET = "\x1b[0m";
+const COLOR_GREEN = "\x1b[32m";
+const COLOR_BLUE = "\x1b[34m";
+const COLOR_RED = "\x1b[31m";
+
+async function checkPluginDocs() {
+  const failureMessages = [];
+  const pluginsPath = "website/data/docs-remote-plugins.json";
+  const pluginsFile = fs.readFileSync(path.join(process.cwd(), pluginsPath));
+  const pluginEntries = JSON.parse(pluginsFile);
+  const entriesCount = pluginEntries.length;
+  console.log(`\nResolving plugin docs from ${entriesCount} repositories …`);
+  for (var i = 0; i < entriesCount; i++) {
+    const pluginEntry = pluginEntries[i];
+    const { title, repo, version } = pluginEntry;
```

Comment:
if one opens a PR with an invalid configuration, for example missing the version, what happens to this script? 
Do you think it makes sense to quickly validate, in this same script, that everything is there (title, path, repo, and version). Don't need to implement it if you think is not a good idea 🤔

Reply:
I think this is a great idea! 💖 Will jump on that now, I think it should be relatively straightforward 👍

Reply:
Implemented a basic validation check, with a dedicated error message, before trying to fetch. Here's some sample output from a fail due to a missing `version`:

<img width="1020" alt="CleanShot 2021-03-20 at 21 47 33@2x" src="https://user-images.githubusercontent.com/4624598/111891059-fb24fd00-89c5-11eb-8bed-5d7a661fa21b.png">

Let me know if this is approx what you had in mind ☺️

Reply:
This looks great! 👏🏼